README.md

```diff
@@ -10,6 +10,10 @@ It is provided here as a convenience to the writers, reviewers and editors of Ma
 The official Mayastor User Reference for the latest GA release can be found [here](https://mayastor.gitbook.io/) 
 {% endhint %}
 
+{% hint style="danger" %}
+This is not the latest version of the documentation. Refer to the [OpenEBS Documentation](https://openebs.io/docs) to view the latest documentation.
```

Comment:
Can we point to the replicated engine overview link?

Reply:
In this message, we also need to specify that Mayastor documentation till version 2.5 are published on this website. For versions 2.6 and above, refer to the openebs docs with the link..

Reply:
> Can we point to the replicated engine overview link?

@avishnu We can do that, but it will point out to the [Installation page of Mayastor](https://openebs.io/docs/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/rs-installation) as it's the 1st page in the Replicated Engine folder.

So kindly suggest, if we can still point out to that or can we point to the [Introduction to OpenEBS page](https://openebs.io/docs/)?

Reply:
Can we say 
Mayastor is now known as OpenEBS Replicated Storage and that the Mayastor latest documentation (v2.6 and above) are now published as part of [OpenEBS documentation](https://openebs.io/docs)?